src/FFmpegWriter.cpp

```diff
@@ -1854,9 +1906,15 @@ void FFmpegWriter::write_audio_packets(bool is_final, std::shared_ptr<openshot::
 			frame_final->nb_samples = audio_input_frame_size;
 
 			// Fill the final_frame AVFrame with audio (non planar)
+#if HAVE_CH_LAYOUT
+			avcodec_fill_audio_frame(frame_final, audio_codec_ctx->ch_layout.nb_channels,
```

Comment:
If possible, let's set a variable for # of channels, so we don't need to repeat the same exact code twice.

Reply:
ok